js/crypto.js

```diff
@@ -156,11 +162,117 @@ function generateJSON(el) {
     el.setAttribute('download', 'Crypto-Touchbar-App-' + selectedFiatObj.ticker + '.json');
 }
 
+function addCustomCoin(customCoin) {
```

Comment:
I think I've seen this coin creation stuff somewhere else in the file, can't we make a generic function for it?

Reply:
Probably yeah, i was being lazy but fair point